mw-webapp/src/dataAccessLogic/PlanForNextPeriodDAL.ts

```diff
@@ -1,11 +1,33 @@
 import {planForNextPeriodToPlanForNextPeriodDTOConverter} from
   "src/dataAccessLogic/BusinessToDTOConverter/planForNextPeriodToPlanForNextPeriodDTOConverter";
+import {DayReportDAL} from "src/dataAccessLogic/DayReportDAL";
 import {planForNextPeriodDTOToPlanForNextPeriodConverter} from
   "src/dataAccessLogic/DTOToBusinessConverter/planForNextPeriodDTOToPlanForNextPeriodConverter";
 import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
 import {TimeUnit} from "src/model/businessModel/time/timeUnit/TimeUnit";
 import {PlanForNextPeriodDTOWithoutUuid, PlanForNextPeriodService} from "src/service/PlanForNextPeriodService";
-import {SPACE} from "src/utils/unicodeSymbols";
+import {unicodeSymbols} from "src/utils/unicodeSymbols";
+
+/**
+ * PlanForNextPeriod props
+ */
+interface PlanForNextPeriodProps {
+
+  /**
+   * PlanForNextPeriod element
+   */
+  planForNextPeriod: PlanForNextPeriod;
+
+  /**
+   * New job of PlanForNextPeriod.job
+   */
+  job?: string;
+
+  /**
+   * New time of PlanForNextPeriod.estimationTime
+   */
+  time?: number;
```

Comment:
why time does not contain measurement value? (minutes)

Reply:
Deleted